lib/html5ever.ex

```diff
@@ -93,26 +115,16 @@ defmodule Html5ever do
 
   """
   def flat_parse(html) do
-    flat_parse_dirty(html)
+    Html5ever.Native.flat_parse(html, false)
   end
 
-  defp parse_dirty(html) do
-    case Html5ever.Native.parse_sync(html) do
```

Comment:
For some reason we were calling the `parse_sync` function here. This may be a performance issue today.